sharepoint-adfs/azuredeploy.json

```diff
@@ -370,13 +370,15 @@
       "spSvcPassword": "[parameters('serviceAccountsPassword')]",
       "spAppPoolUserName": "spapppool",
       "spAppPoolPassword": "[parameters('serviceAccountsPassword')]",
-      "spPassphrase": "[parameters('serviceAccountsPassword')]"
+      "spPassphrase": "[parameters('serviceAccountsPassword')]",
+      "localAdminUserName": "[concat('local-', parameters('adminUserName'))]"
```

Comment:
how do users know the local user name?

Reply:
in SharePoint, admins always want to sign-in as the domain account, using the local admin account is causing only confusion.
But when doing RDP, many admins set the user as "accountName" instead of "domain\accountName", which signs-in as the local admin (and it's really always undesired).
With this change, RDP will validate sign-in only if the admin explicitly set user as "domain\accountName", which I expect it to be a logical test if the attempt as "accountName" fails. Should I add a note in the README?

Reply:
It seems like we should have 1 of 2 things... either we output the value so the user knows what it is (adding to readme is probably ok too) or we generate a random user name since we never expect them to use it.  

If it's a case where they could use the local account but shouldn't, then output and readme makes sense.  If it's a case where they should never user it, then random makes more sense.

Reply:
This is a great suggestion, I didn't think about this.
Looks like [function uniqueString](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-string?tabs=json#uniquestring) is the way to go.
I'm testing it now, if/when it works I'll push the commit